integrations/access/jira/app.go

```diff
@@ -290,6 +290,9 @@ func (a *App) onWatcherEvent(ctx context.Context, event types.Event) error {
 
 // onJiraWebhook processes Jira webhook and updates the status of an issue
 func (a *App) onJiraWebhook(ctx context.Context, webhook Webhook) error {
+	ctx, cancel := context.WithTimeout(ctx, webhookIssueAPIRetryTimeout)
```

Comment:
This context will still be canceled when the parent context is canceled, is that what you intend?

Reply:
Ah yes, Zac is right, I don't think the fix will work right now. We need a separate context here.

Reply:
I think that should be fine if that only happens when the cancel func of the parent or child is called. As long as this updates the timeout correctly I think[ it should be fine](https://go.dev/play/p/Oz4d1U8qSDq)

Reply:
Swapped to using the background context instead of the parent one.